### ISSUE TITLE
Provide tarantool 1.5 formula for Homebrew

### ISSUE THREAD

**Issue by vmarkovtsev** [tarantool/tarantool#1497]
Since Mail.Ru servers are using 1.5, I need to install that version on my Mac. Homebrew provides only the latest 1.6 branch at the moment, and there is no versions/tarantool/1.5 => I have to compile it from scratch. So sad. I suggest to add 1.5 or convince Mail.Ru admins to switch to 1.6 once and for all.

**Comment by bigbes** [comment]
You can build 1.5 on Mac? As I know, Mac OS X is not supported.

**Comment by bigbes** [comment]
If you can build it, then you may use `brew install https://github.com/tarantool/tarantool/blob/stable/extra/tarantool.rb` command to install Tarantool using Homebrew

**Comment by vmarkovtsev** [comment]
Well, there is README.MacOSX in 1.5.5 with some instructions, I believe it is supported.

UPD: it seems that there are some minor clang incompatibilities, I will see about them.

**Comment by vmarkovtsev** [comment]
I was able to successfully build and install Tarantool after applying 2 patches.
```ruby
    ohai "Preparing"
    version = `git -C #{cached_download} describe HEAD`

    File.open(buildpath/"VERSION", 'w') {|file| file.write(version)}
    system "patch", "-p0", "third_party/libeio/ecb.h", "01_ecb.patch"
    system "patch", "-p0", "third_party/libev/ev.c", "02_ev.patch"
```

01_ecb.patch
```
--- third_party/libeio/ecb.h	2016-05-23 16:41:55.000000000 +0300
+++ third_party/libeio/ecb.h.patched	2016-05-23 16:41:43.000000000 +0300
@@ -82,7 +82,7 @@
 
 #define ECB_C     (__STDC__+0) /* this assumes that __STDC__ is either empty or a number */
 #define ECB_C99   (__STDC_VERSION__ >= 199901L)
-#define ECB_C11   (__STDC_VERSION__ >= 201112L)
+#define ECB_C11   0  // (__STDC_VERSION__ >= 201112L)
 #define ECB_CPP   (__cplusplus+0)
 #define ECB_CPP11 (__cplusplus >= 201103L)
```

02_ev.patch
```
--- third_party/libev/ev.c	2015-04-22 19:43:04.000000000 +0300
+++ third_party/libev/ev.c.patched	2016-05-23 16:50:11.000000000 +0300
@@ -559,7 +559,7 @@
 
 #define ECB_C     (__STDC__+0) /* this assumes that __STDC__ is either empty or a number */
 #define ECB_C99   (__STDC_VERSION__ >= 199901L)
-#define ECB_C11   (__STDC_VERSION__ >= 201112L)
+#define ECB_C11   0  // (__STDC_VERSION__ >= 201112L)
 #define ECB_CPP   (__cplusplus+0)
 #define ECB_CPP11 (__cplusplus >= 201103L)
```

**Comment by vmarkovtsev** [comment]
To use it smoothly, I also had to change the following in tarantool.rb
```
--- tarantool.rb	2016-05-23 17:18:53.000000000 +0300
+++ tarantool.rb.patched	2016-05-23 17:18:41.000000000 +0300
@@ -50,6 +50,8 @@
     version = `git -C #{cached_download} describe HEAD`
 
     File.open(buildpath/"VERSION", 'w') {|file| file.write(version)}
+    system "patch", "-p0", "third_party/libeio/ecb.h", "01_ecb.patch"
+    system "patch", "-p0", "third_party/libev/ev.c", "02_ev.patch"
 
     ohai "Configuring:"
     system "cmake", ".", *args
@@ -59,6 +61,8 @@
 
     ohai "Installing:"
     system "make install"
+    system "mkdir", "-p", "#{prefix}/var/run"
+    system "mkdir", "-p", "#{prefix}/var/lib/tarantool"
 
     ohai "Installing man"
     man1.install 'doc/man/tarantool.1'
@@ -69,6 +73,7 @@
     ohai "Installing config"
     if build.stable?
       inreplace prefix/"etc/tarantool.cfg", /^work_dir =.*/, "work_dir = #{prefix}/var/lib/tarantool"
+      inreplace prefix/"etc/tarantool.cfg", /^pid_file =.*/, "pid_file = #{prefix}/var/run/box.pid"
     else
       doc.install "test/box/box.lua"
       inreplace doc/"box.lua" do |s|
```

**Comment by bigbes** [comment]
Applied your patches with some fixes. Works now. Thanks.
